torch/csrc/jit/codegen/cuda/ir_base_nodes.cpp

```diff
@@ -63,9 +63,7 @@ Val::Val(const Val* src, IrCloner* ir_cloner)
       vtype_(src->vtype_),
       dtype_(src->dtype_),
       is_fusion_input_(src->is_fusion_input_),
-      is_fusion_output_(src->is_fusion_output_),
-      definition_(src->definition_),
-      uses_(src->uses_) {}
+      is_fusion_output_(src->is_fusion_output_) {}
```

Comment:
The comment above, "we copy the original pointers", does not seem to reflect what the actual code does.

Reply:
Please let me know which part doesn't match the comment. The copying was first copy the nodes, and the use-defs are resolved at later stage of fusion::copy here:
[https://github.com/csarofeen/pytorch/blob/multi_kernel_merged/torch/csrc/jit/codegen/cuda/fusion.cpp#L93-L96](https://github.com/csarofeen/pytorch/blob/multi_kernel_merged/torch/csrc/jit/codegen/cuda/fusion.cpp#L93-L96)

Reply:
`definition_` and `uses_` were set before but not now. I thought the comment was referring to copying like `definition_(src_definition_)`.

Reply:
The comment meant exactly not copying `definition_` and `uses_`. So I guess code wasn't matching with comment before but actually matches after.

Reply:
They were copied at `definition_(src->definition)` and `uses_(src->uses_)`. They are not now, so the comment should be updated to avoid confusion.

Reply:
Added to the comment. Thanks!